src/main/java/teammates/common/datatransfer/StudentAttributes.java

```diff
@@ -384,4 +384,30 @@ public void setCreated_NonProduction(Date createdAt) {
     public void setUpdatedAt_NonProduction(Date updatedAt) {
         this.updatedAt = updatedAt;
     }
+
+    /**
+     * Checks whether the edit form of student has changed the section value.
+     * @param originalStudentAttribute
+     * @return true if either section has changed from previous values. 
```

Comment:
Could change this to "@return true if section has been changed from its original value."